engine/src/main/scala/cromwell/webservice/metadata/MetadataComponent.scala

```diff
@@ -0,0 +1,50 @@
+package cromwell.webservice.metadata
+
+import cats.{Monoid, Semigroup}
+import cats.instances.map._
+import cromwell.core.{ExecutionStatus, WorkflowState}
+import spray.json.{JsArray, _}
+
+object MetadataComponent {
+  implicit val MetadataComponentMonoid: Monoid[MetadataComponent] = new Monoid[MetadataComponent] {
+    private lazy val stringValueMapSg = implicitly[Semigroup[Map[String, MetadataComponent]]]
+    private lazy val intValueMapSg = implicitly[Semigroup[Map[Int, MetadataComponent]]]
```

Comment:
these seem like they should be named `stringKeyMapSg` and `intKeyMapSg` respectively